artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -272,7 +272,7 @@ public class Create extends InstallAbstract {
    @Option(names = "--jdbc", description = "Store message data in JDBC instead of local files.")
    boolean jdbc;
 
-   @Option(names = "--staticCluster", description = "Cluster node connectors list separated by comma, e.g. \"tcp://server:61616,tcp://server2:61616,tcp://server3:61616\".")
+   @Option(names = {"--staticCluster", "--static-cluster"}, description = "Cluster node connectors list separated by comma, e.g. \"tcp://server:61616,tcp://server2:61616,tcp://server3:61616\".")
```

Comment:
Feels like it should be a seperate change, has little directly to do with moving+improving this test, and folks are less likely to know it was added if buried in this commit.

Reply:
It could still have been in the same PR if needed but its clearly a seperate change that should have a commit+JIRA of its own rather than going in with the otherwise unrelated test-only changes.